sdk/src/Services/S3/Custom/Transfer/Internal/_async/MultipartUploadCommand.async.cs

```diff
@@ -269,7 +270,7 @@ private void AbortMultipartUpload(string uploadId)
 
                     this._uploadResponses = uploadPartResponses;
                     CompleteMultipartUploadRequest compRequest = ConstructCompleteMultipartUploadRequest(initiateResponse, true, requestEventHandler);
-                    await _s3Client.CompleteMultipartUploadAsync(compRequest).ConfigureAwait(false);
+                    await _s3Client.CompleteMultipartUploadAsync(compRequest, cancellationToken).ConfigureAwait(false);
                     Logger.DebugFormat("Completed multi part upload. (Part count: {0}, Upload Id: {1})", uploadPartResponses.Count, initiateResponse.UploadId);
                 }
             }
```

Comment:
Just for reference, it's fine to not to add `CancellationToken` in `AbortMultipartUploadAsync()`. If the cancellation happens before previous `InitiateMultipartUploadAsync()` is cancelled, then this `catch()` block is not reached. Whereas, if cancellation happens after `InitiateMultipartUploadAsync()`, then this exception block would be reached and we should abort multipart upload.